packages/security/src/strategy/scoring.ts

```diff
@@ -21,99 +21,99 @@ export const SCORING_WEIGHTS = {
 export const complexityScore = (complexity: StrategyComplexity) => {
   switch (complexity) {
     case StrategyComplexity.LOW:
-      return 1;
+      return 10;
```

Comment:
Why not leave it at 0-1 and just mutliply the resulting value in the frontend?

I think this would be better, if we ever decide to change it to 0-100 or something. 🤷

Reply:
makes sense

Reply:
Fixed